src/main/java/io/stargate/sgv2/jsonapi/exception/mappers/ConstraintViolationExceptionMapper.java

```diff
@@ -76,12 +77,24 @@ private static String valueDescription(Object rawValue) {
       return "<JSON value of " + rawValue.toString().length() + " characters>";
     }
 
+    if (rawValue instanceof FindCommand) {
+      if (propertyPath.contains("options.skip")) {
+        return String.format("%s", ((FindCommand) rawValue).options().skip());
+      }
+      if (propertyPath.contains("options.limit")) {
+        return String.format("%s", ((FindCommand) rawValue).options().limit());
+      }
+      if (propertyPath.contains("options.pageState")) {
+        return String.format("%s", ((FindCommand) rawValue).options().pageState());
+      }
+    }
+
     String valueDesc = rawValue.toString();
     if (valueDesc.length() <= MAX_VALUE_LENGTH_TO_INCLUDE) {
-      return String.format("\"%s\"", valueDesc);
+      return String.format("%s", valueDesc);
     }
```

Comment:
The return JSON body will contain the escape character `\`
```
{
    "errors": [
        {
            "message": "Request invalid: field 'command.options.skip' value \"FindCommand[filterClause=null, projectionDefinition=null, sortClause=null, options=Options[limit=null, skip=1, pageState=null, includeSimilarity=false]]\" not valid. Problem: skip options should be used with sort clause.",
            "errorCode": "COMMAND_FIELD_INVALID"
        }
    ]
}
```
so I removed it

Reply:
Oh, some tests include this character `\`. @tatu-at-datastax Do we add `\` intentionally?

Reply:
Those quotes were there on purpose, to indicate `String` value. JSON will need to have escapes, that is fine.
We could use single quotes instead, but it is nice to know type of value and sometimes to separate it from message itself (without quotes it is sometimes hard to tell what is value, what is message).

Backslash is needed/added by JSON on serialization, but Java source code also requires escaping.
We do not add it otherwise than those.

Reply:
I understand we need to use quotes to separate the value from messages. But using double quotes will need a backslash to escape and the backslash will somehow show in the return body. I think it would be better to use single quotes to avoid the backslash appearing in the return body?

Reply:
Backslash is needed for double-quote if included in JSON; it is not added by us (and will be "removed" when reading JSON by code) but handled automatically.
But I'm fine going with single-quote to make it easier to read when enclosed in JSON.